omod/src/main/resources/apps/aijar_homepage_app.json

```diff
@@ -24,7 +24,8 @@
         "label": "aijar.app.backupdatabase.list.label",
         "url": "module/databasebackup/backup.form",
         "icon": "icon-cloud-download",
-        "order": 115
+        "order": 115,
+        "requiredPrivilege": "App: aijar.backupDatabase"
```

Comment:
@slubwama Better to add this in the database Backup module

Reply:
@ssmusoke That means that the app it self is introduced in the reporting module rather than here.

Reply:
Better to have the apps and privileges in the respective modules so that they are easily reusable and consistent. Even if we re the only ones who will be using them.

Reply:
@ssmusoke the Backup module doesn't have this app implemented in it yet there is an instance of the app in our implementation. What is being done here is adding permissions to all apps that being implemented or overridden in our implementation/distribution. Unless Backup module has the app. The restriction will keep on our side.